ygen/gogen.go

```diff
@@ -1379,6 +1392,17 @@ func writeGoStruct(targetStruct *Directory, goStructElements map[string]*Directo
 		YANGPath:   util.SlicePathToString(targetStruct.Path),
 	}
 
+	// Non-fakeroot GoStructs all have a module to which they belong per
+	// RFC7950 namespace rules. The fakeroot is assigned "".
+	if !IsFakeRoot(targetStruct.Entry) {
+		im, err := targetStruct.Entry.InstantiatingModule()
+		if err != nil {
```

Comment:
Should this be fatal? If there is an entry that is defined that doesn't have an instantiating module, then this seems problematic - such that we can never actually render this correctly according to 7951.

Reply:
Suggest we address this by just returning the slice of errors here.

Reply:
I believe all nodes in YANG belong to a namespace, so I think it's not possible for this to return an error if everything is working properly: https://datatracker.ietf.org/doc/html/rfc7950#section-6.2.1

SGTM with returning the slice of errors directly.

Reply:
Done.